gradle/libs.versions.toml

```diff
@@ -54,6 +54,8 @@ androidx-compose-material3-windowSizeClass = { module = "androidx.compose.materi
 androidx-compose-runtime = { module = "androidx.compose.runtime:runtime" }
 androidx-compose-runtime-livedata = { module = "androidx.compose.runtime:runtime-livedata" }
 androidx-compose-ui = { module = "androidx.compose.ui:ui" }
+# https://androidx.tech/artifacts/compose.ui/ui-util/
```

Comment:
can we link to the officiel docs instead?
https://developer.android.com/reference/kotlin/androidx/compose/ui/text/googlefonts/package-summary

If they start to add written docs (like in [room](https://developer.android.com/reference/kotlin/androidx/room/package-summary) for instance) we would not be able to view them.